.dockerignore

```diff
@@ -0,0 +1,2 @@
+*
+!docker/
```

Comment:
Yeah, I got just confused for a moment! I wanted to ignore this file, but this commit actually adds it to git, that's why I said "Oh, nevermind" (seems that that comment is gone now!)